### ISSUE TITLE
[COGNITION] 508-defect-3: Appointment tabs SHOULD have an accessible label

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#15052]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

As a screen reader user, I want to hear tab labels that answer Upcoming...what? Past...what? Express Care...what? I understand these are appointments, but if I'm navigating by links or have a cognitive issue like short-term memory loss, I might not remember this is an appointments page.

## Details

The appointment tabs (Upcoming, Past, Express Care) would benefit from aria-label attributes. This will help screen reader users and those who might also have cognitive issues understand what Upcoming, Past, Express Care represent.

## Acceptance Criteria

- [ ] All appointment tabs have an aria-label that contains the visual text "Upcoming | Past | Express Care" + " appointments"
- [ ] Zero axe browser scan violations are detected

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Environment

* https://staging.va.gov/health-care/schedule-view-va-appointments/appointments/
* https://staging.va.gov/health-care/schedule-view-va-appointments/appointments/past
* https://staging.va.gov/health-care/schedule-view-va-appointments/appointments/express-care

## Code Fix (if known)
```diff
! Upcoming Appointments
! Will need an aria-label for Past and Express Care appointment tabs as well

<a
+ aria-label="Upcoming appointments"
  aria-current="page"
  class="vaos-appts__tab vads-u-text-align--center vads-u-color--gray-dark vaos-appts__tab--current"
  id="tabupcoming"
  aria-selected="false"
  role="tab"
  tabindex="-1"
  href="/health-care/schedule-view-va-appointments/appointments/"
>Upcoming</a>
```

## WCAG or Vendor Guidance (optional)

* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

## Screenshots or Trace Logs

![Screen Shot 2020-10-20 at 10 37 42 AM](https://user-images.githubusercontent.com/934879/96609827-8d629100-12c0-11eb-8a34-c26729dbf841.png)

**Comment by grossb1** [comment]
@1Copenut This has been deployed to staging and is ready to test.

**Comment by 1Copenut** [comment]
This is behaving well, moving to close.

One thing to note, that I'd like to get user feedback on if possible. The tabs don't announce themselves as links in the screen readers. Where they have ARIA `role="tab"` they behave as controls, and don't have the same affordances as links. If we could schedule some remote screen reader research, even 3-5 participants, to learn more about this feature, I think that would be beneficial. I'll circle back properly on Monday.

/cc @lenaecb && @rtwell

**Comment by laurenernest** [comment]
@1Copenut Noted. Thanks for the review!
